### PR TITLE
gh-1244 renamings

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift [hunk 1]
```diff
@@ -117,7 +117,7 @@ enum TrackingEvent: String, Trackable {
 
     case chooseOwner                                = "screen_owner_choose"
     case bannerImportOwnerKeySkipped                = "user_banner_owner_skip"
-    case bannerImportOwnerKeyImported               = "user_banner_owner_import"
+    case bannerImportOwnerKeyAdd                    = "user_banner_owner_add"
```
Comment (DmitryBespalov) [inline Multisig/Cross-layer/Analytics/TrackingEvent.swift:120]: I would not change tracking because it would invalidate previous tracking

Comment (DmitryBespalov) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift:120]: I mean tracking string value - the variable change is ok

**Review by DmitryBespalov — APPROVED** [review]
good, just one comment

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift (hunk 1 above)
Comment (sche) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift:120]: this was requested in the bug description. I will ask there one more time

Comment (sche) [reply Multisig/Cross-layer/Analytics/TrackingEvent.swift:120]: The change was requested during refinement meeting

**Pull request merged**